migtests/scripts/live-migration-run-test.sh

```diff
@@ -37,6 +37,20 @@ else
 	source ${SCRIPTS}/${SOURCE_DB_TYPE}/env.sh
 fi
 
+export EXPORT_DIR=${EXPORT_DIR:-"${TEST_DIR}/${NORMALIZED_TEST_NAME}_live_export-dir"}
```

Comment:
Looks like this code is the same for all tests? 
if so, can we write this logic in functions.sh? (for determining exportDir, setting SOURCE_DB_NAME, etc.